codes/c/chapter_computational_complexity/time_complexity.c

```diff
@@ -0,0 +1,192 @@
+// File: time_complexity.c
+// Created Time: 2023-01-03
+// Author: sjinzh (sjinzh@gmail.com)
```

Comment:
Please add the code to the docs (markdown file).

Reply:
done.

Reply:
> Please add the code to the docs (markdown file).

done.